go/client/cmd_pgp_gen.go

```diff
@@ -68,6 +73,32 @@ func (v *CmdPGPGen) Run() (err error) {
 	return err
 }
 
+// Duplicates the test in libkb.PGPGenArg.CreatePGPIDs() for interactive input
+var CheckPGPID = libkb.Checker{
+	F: func(s string) bool {
+		if !strings.Contains(s, "<") && libkb.CheckEmail.F(s) {
+			return true
+		}
+		_, err := libkb.ParseIdentity(s)
+		return err == nil
+	},
+	Hint: "2-12 letter id or email, or pgp style id",
```

Comment:
Yeah that prompt could be better but that's what I found: an individual ID can be entered as an email address without angle brackets, which we will parse and assign to the email field of the ID. This is separate from optional pgpID parsing So it is something like:
((name | email) | pgpID)
where pgpID is:
name (comment) <email>
  ... where (comment) is optional and at least one of name or email is specified.
Would it be simpler to skip the pgp parsing? Maybe that's intended and I missed it?